data/data/agent/files/usr/local/bin/load-config-iso.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+
+AGENT_CONFIG_ARCHIVE_FILE="config.gz"
+AGENT_CONFIG_MOUNT="/media/config-image"
+CLUSTER_IMAGE_SET="/etc/assisted/manifests/cluster-image-set.yaml"
+AGENT_CONFIG_MOUNT="/media/config_image"
```

Comment:
Duplicated var? Line 6 is redundant I guess?

Reply:
Yes, thanks.